tests/examples/example_SBT_Lo_T.out

```diff
@@ -0,0 +1,224 @@
+                               *****************
+                               ***CASE REPORT***
+                               *****************
+
+Simulation Metadata
+----------------------
+ GEOPHIRES Version: 3.5.2
+ Simulation Date: 2024-09-02
+ Simulation Time:  14:35
+ Calculation Time:     45.642 sec
+
+                           ***SUMMARY OF RESULTS***
+
+      End-Use Option: Electricity
+      Average Net Electricity Production:                     0.00 MW
+      Electricity breakeven price:                        18339.44 cents/kWh
```

Comment:
Is this the correct/intended result for this output?

Reply:
yes, the Lo-T version is just to validate the reservoir model, not the economics (as it says in the paper).